pkg/templates/kubeadm/v1beta3/kubeadm.go

```diff
@@ -177,17 +178,29 @@ func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, er
 		APIServer: kubeadmv1beta3.APIServer{
 			ControlPlaneComponent: kubeadmv1beta3.ControlPlaneComponent{
 				ExtraArgs: map[string]string{
-					"endpoint-reconciler-type": "lease",
-					"service-node-port-range":  cluster.ClusterNetwork.NodePortRange,
-					"enable-admission-plugins": kubeflags.DefaultAdmissionControllers(kubeSemVer),
+					"enable-admission-plugins":      kubeflags.DefaultAdmissionControllers(kubeSemVer),
+					"endpoint-reconciler-type":      "lease",
+					"kubelet-certificate-authority": "/etc/kubernetes/pki/ca.crt",
+					"profiling":                     "false",
+					"request-timeout":               "300s",
+					"service-node-port-range":       cluster.ClusterNetwork.NodePortRange,
+					"tls-cipher-suites":             safeTLSCiphers(),
 				},
 				ExtraVolumes: []kubeadmv1beta3.HostPathMount{},
 			},
 			CertSANs: certSANS,
 		},
 		ControllerManager: kubeadmv1beta3.ControlPlaneComponent{
 			ExtraArgs: map[string]string{
-				"flex-volume-plugin-dir": "/var/lib/kubelet/volumeplugins",
+				"flex-volume-plugin-dir":      "/var/lib/kubelet/volumeplugins",
+				"profiling":                   "false",
+				"terminated-pod-gc-threshold": "100",
```

Comment:
I'm kinda confused with this parameter. The [upstream default value](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/) is 12500 but it seems ridiculously high. How did we come up with 100?

>  --terminated-pod-gc-threshold int32     Default: 12500
>
> Number of terminated pods that can exist before the terminated pod garbage collector starts deleting terminated pods. If <= 0, the terminated pod garbage collector is disabled.

Reply:
Report
```
[WARN] 1.3.1 Ensure that the --terminated-pod-gc-threshold argument is set as appropriate (Manual)

1.3.1 Edit the Controller Manager pod specification file /etc/kubernetes/manifests/kube-controller-manager.yaml
on the control plane node and set the --terminated-pod-gc-threshold to an appropriate threshold,
for example, --terminated-pod-gc-threshold=10
```

But 10, I thought it, is too small. So I just took a sensible guess.

Reply:
Same as above, my understanding of that report is that they provided an example how to set the flag. I agree that 10 is too small, but I think we should use the same value as kube-controller-manager by default.

Reply:
I don't get it, "ridiculously high" and "use the default" are mutually exclusive. We have to pick one. Default: 12500 means that 12500 pod specs could potentially litter etcd storage for no good reason. Let's pick something smaller, 500? 1000? whatever, but not 12500 :exploding_head:

Reply:
Let's take 20k per pod, that's very roughly 230Mb (12500 * 20kb) of excess bytes in etcd.

Reply:
Updated to 1000.